### ISSUE TITLE
Nested marshmellow schemas

### ISSUE THREAD

**Issue by edkb** [apryor6/flask_accepts#24]
This lib is really nice and works great with simple marshmallow schemas. Unfortunately Swagger doesn't display correctly when using nested schemas like [this](https://marshmallow.readthedocs.io/en/stable/nesting.html)

**Comment by apryor6** [comment]
There’s no reason for it not to work, so it must be a bug. I’ll look into it, thanks!

**Comment by apryor6** [comment]
The default values are not currently being passed through, so that is a relatively simple change for me to make within `flask_accepts`.

However, it appears that default parameters in the body are not being used in flask-RESTplus. I've got an open discussion upstream with them. 

Both of these will need to be resolved, and then this should work.

**Comment by apryor6** [comment]
I should add that nested Schemas work as far as serialization goes. It’s only the documentation that is lacking, as it just defaults to empty curly braces for the nested object.

**Comment by edkb** [comment]
Nice, I've open this issue here because I had problems to make this work with RESTplus too and the auto-docs is one of the main reasons to use this framework. Hope its a simple fix

**Comment by edkb** [comment]
I've found another strange thing, I don't know if it is related to this (I may open another issue if necessary)  but swagger is showing a required field called payload that I didn't specified in the code.

```
from flask import Blueprint
from flask_restplus import Api, Resource
from flask_accepts import accepts, responds
from marshmallow import Schema, fields

blueprint = Blueprint('api', __name__)

api = Api(
    app=blueprint,
    title='API',
    version='1.0',
    description='api web service'
)


class UserSchema(Schema):
    name = fields.String()
    email = fields.Email()
    created_at = fields.DateTime()


@api.route("/widget")
class WidgetResource(Resource):
    @accepts(schema=UserSchema(many=True), api=api)
    @responds(schema=UserSchema(many=True), api=api)
    def get(self):
        return 200
```
![image](https://user-images.githubusercontent.com/12564798/64136545-9fd1f080-cdc8-11e9-998c-d2db46c5858f.png)

**Comment by apryor6** [comment]
For `api.model`, it turns out the proper keyword for modifying the default value is `example` and NOT `default`. I'll just need to create a mapping from marshmallow `default` to this `example` param (with a default value if one was not provided), and this should be good to go

**Comment by apryor6** [comment]
Thanks to @j5awry for the help with that

**Comment by apryor6** [comment]
Thanks for bringing this up, should now be fixed. Let me know if there are still issues.

**Comment by edkb** [comment]
Thanks, it worked! But there there's still is one more small thing. It uses a default name for the nested models and the correct name just for the root, so I think it should display correct name for the nested models too.

Example: When I use the model Cog nested in the Widget(POST), swagger shows "DefaultResponseModel_2". But when I use just the Cog not as a nested model (GET) it shows it's correct name.  

```
class CogSchema(Schema):
    cog_foo = fields.String(default="cog")
    cog_baz = fields.Integer(default=999)


class WidgetSchema(Schema):
    foo = fields.String(default="test string")
    baz = fields.Integer(default=42)
    flag = fields.Bool(default=False)
    date = fields.Date(default="01-01-1900")
    dec = fields.Decimal(default=42.42)
    dct = fields.Dict(default={"key": "value"})

    cog = fields.Nested(CogSchema)


@api.route("/restplus/make_a_widget")
class WidgetResource(Resource):
    
    @accepts(dict(name="some_arg", type=str), schema=CogSchema, api=api)
    @responds(schema=CogSchema, api=api)
    def get(self):
        from flask import jsonify

        return request.parsed_obj
    
    @accepts(dict(name="some_arg", type=str), schema=WidgetSchema, api=api)
    @responds(schema=WidgetSchema, api=api)
    def post(self):
        from flask import jsonify

        return request.parsed_obj
```
![Screenshot_2019-09-04 API](https://user-images.githubusercontent.com/12564798/64257430-baf14d00-cefb-11e9-9d2f-c443a3d236e5.png)

**Comment by apryor6** [comment]
Thanks for such careful testing -- this will be a simple change

**Comment by apryor6** [comment]
This should be fixed in `0.11.2`. Give it a try.

**Comment by edkb** [comment]
Thanks AJ, it works perfectly!
